docs/rst/knora-api-server/api_v2/ontology-information.rst

```diff
@@ -18,10 +18,1077 @@
+at once. In most cases, an update request is a JSON-LD document containing ``knora-api:hasOntologies``,
+providing only the information that is relevant to the update.
+
+Moreover, the API enforces the following rules:
+
+- An entity (i.e. a class or property) cannot be referred to until it has been created.
+
+- An entity cannot be modified or deleted if it is used in data, except for changes to its
+  ``rdfs:label`` or ``rdfs:comment``.
+
+- An entity cannot be modified if another entity refers to it, with one exception: a ``knora-api:subjectType`` or
+  ``knora-api:objectType`` that refers to a class will not prevent the class's cardinalities from being modified.
+
+Because of these rules, some operations have to be done in a specific order:
+
+- Properties have to be defined before they can be used in the cardinalities of a class,
+  but a property's ``knora-api:subjectType`` cannot refer to a class that does not yet exist. The recommended
+  approach is to first create a class with no cardinalities, then create the properties that it needs,
+  then add cardinalities for those properties to the class.
+
+- To delete a class along with its properties, the client must first remove the cardinalities
+  from the class, then delete the property definitions, then delete the class definition.
+
+When changing an existing ontology, the client must always supply the ontology's ``knora-api:lastModificationDate``,
+which is returned in the response to each update. If user A attempts to update an ontology, but user B
+has already updated it since the last time user A received the ontology's ``knora-api:lastModificationDate``,
+user A's update will be rejected with an HTTP 409 Conflict error. This means that it is possible for two different
+users to work concurrently on the same ontology, but this is discouraged since it is likely to lead to confusion.
+
+An ontology can be created or updated only by a system administrator, or by a project administrator in the
+ontology's project.
+
+Ontology updates always use the default schema.
+
+Creating a New Ontology
+^^^^^^^^^^^^^^^^^^^^^^^
+
+An ontology is always created within a particular project. This is the only ontology update request in which
+the client submits a JSON-LD document that does not contain ``knora-api:hasOntologies``.
+
+::
+
+    HTTP POST to http://host/v2/ontologies
+
+.. code-block:: jsonld
+
+   {
+       "knora-api:ontologyName": "ONTOLOGY_NAME",
+       "knora-api:projectIri": "PROJECT_IRI",
+       "rdfs:label": "ONTOLOGY_LABEL",
+       "@context": {
+           "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+           "knora-api": "http://api.knora.org/ontology/knora-api/v2#"
+       }
+   }
+
+A successful response will be a JSON-LD document containing ``knora-api:hasOntologies``,
+providing only the ontology's metadata.
+
+Changing an Ontology's Metadata
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Currently, the only modifiable ontology metadata is the ontology's ``rdfs:label``.
+
+::
```

Comment:
``knora-api:hasOntologies`` is not used here either (see "Creating a New Ontology")